src/reducers/search.js

```diff
@@ -38,6 +38,7 @@ export const initialState = {
     displayName: 'Top inspiration',
     id: 'marketing:homepage.dk.spies'
   },
+  scrollPage: 6,
```

Comment:
No, I was just curious because it wasn't obvious. 

It might be good to parameterise out the 30, 6 and 5 to somewhere common though. They're spread out in a few different places and have dependencies on one another which feels a bit fragile at the moment.